osu.Game/Overlays/BeatmapSet/ExplicitBeatmapPill.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Overlays.BeatmapSet
+{
+    public class ExplicitBeatmapPill : CompositeDrawable
+    {
+        public ExplicitBeatmapPill()
+        {
+            AutoSizeAxes = Axes.Both;
+        }
+
+        [BackgroundDependencyLoader(true)]
+        private void load(OsuColour colours, OverlayColourProvider colourProvider)
+        {
+            InternalChild = new CircularContainer
+            {
+                Masking = true,
+                AutoSizeAxes = Axes.Both,
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = colourProvider?.Background5 ?? colours.Gray2,
+                    },
+                    new OsuSpriteText
+                    {
+                        Margin = new MarginPadding { Horizontal = 10f, Vertical = 2f },
+                        Text = "EXPLICIT",
+                        Font = OsuFont.GetFont(size: 10, weight: FontWeight.Bold),
```

Comment:
I believe this should actually be
```suggestion
                        Font = OsuFont.GetFont(size: 10, weight: FontWeight.SemiBold),
```

![2021-01-13-210939_341x75_scrot](https://user-images.githubusercontent.com/20418176/104505099-b060c080-55e3-11eb-8372-db568d9eecd1.png)

Reply:
Right, got lost in the web trying to figure that out.